tillerinobot/src/main/java/tillerino/tillerinobot/lang/Spanish.java

```diff
@@ -0,0 +1,217 @@
+
+import java.util.List;
+import java.util.Random;
+
+import org.tillerino.osuApiModel.Mods;
+import org.tillerino.osuApiModel.OsuApiUser;
+
+import tillerino.tillerinobot.BeatmapMeta;
+import tillerino.tillerinobot.IRCBot.IRCBotUser;
+import tillerino.tillerinobot.RecommendationsManager.Recommendation;
+
+/**
+ * @author Sibchil https://osu.ppy.sh/u/3988045 https://github.com/sibchil
+ */
+public class Spanish implements Language {
+
+	@Override
+	public String unknownBeatmap() {
+		return "Lo siento, no conozco ese mapa. Es muy reciente, muy dif�cil, unranked o modo de osu! no est�ndar.";
+	}
+
+	@Override
+	public String internalException(String marker) {
+		return "Ehh... Parece que Tillerino humano estrope� mi cableado."
+		                + " Si no se da cuenta pronto, �podr�as [https://github.com/Tillerino/Tillerinobot/wiki/Contact informarle]? (c�digo de referencia "
+		                + marker + ")";
+	}
+
+	@Override
+	public String externalException(String marker) {
+		return "�Qu� est� pasando? Solo recibo disparates del servidor de osu. �Podr�as decirme que se supone que significa esto? 0011101001010000"
+		                + " Tillerino humano dice que no hay nada por qu� preocuparse y que deber�amos intentarlo de nuevo."
+		                + " Si est�s preocupado por alguna raz�n, puedes [https://github.com/Tillerino/Tillerinobot/wiki/Contact comunic�rselo]. (c�digo de referencia "
+		                + marker + ")";
+	}
+
+	@Override
+	public String noInformationForModsShort() {
+		return "no hay datos para los mods solicitados";
+	}
+
+	@Override
+	public void welcomeUser(IRCBotUser user, OsuApiUser apiUser, long inactiveTime) {
+		if(inactiveTime < 60 * 1000) {
+			user.message("beep boop");
+		} else if(inactiveTime < 24 * 60 * 60 * 1000) {
+			user.message("Bienvenid@, " + apiUser.getUserName() + ".");
+		} else if(inactiveTime > 7l * 24 * 60 * 60 * 1000) {
+			user.message(apiUser.getUserName() + "...");
+			user.message("...�eres t�? �Ha pasado tanto tiempo!");
+			user.message("Es genial tenerte de vuelta. �Puedo ofrecerte una recomendaci�n?");
+		} else {
+			String[] messages = {
+					"me parece que necesitas una recomendaci�n.",
+					"�c�mo me alegro de verte! :)",
+					"mi humano favorito. (�No se lo digas a los otros humanos!)",
+					"�que agradable sorpresa! ^.^",
+					"Ten�a la esperanza de que te dejar�as ver. Todos los dem�s humanos son pat�ticos, �pero no les cuentes lo que dije! :3",
+					"�qu� te apetece hacer hoy?",
+			};
+			
+			Random random = new Random();
+			
+			String message = messages[random.nextInt(messages.length)];
+			
+			user.message(apiUser.getUserName() + ", " + message);
+		}
+	}
+
+	@Override
+	public String unknownCommand(String command) {
+		return "comando desconocido \"" + command
+				+ "\". �Escribe !help si necesitas ayuda!";
+	}
+
+	@Override
+	public String noInformationForMods() {
+	    return "Lo siento, no puedo proporcionar informaci�n sobre esos mods ahora mismo.";

```

Comment:
Shouldn't it be `esas`? We refer to mods as a female here in italian (e.g. "queste mod" -> "esas mods" and not "questi mod" -> "esos mods"). It's probably normal in the spanish osu!community to say esos, but I just want to be sure you didn't make a mistake here :P